functools_lru_cache-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,18 @@
+{% set name = "functools_lru_cache" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+requirements:
+  run:
+    - backports.functools_lru_cache =={{ version }}
```

Comment:
The == are redundant at best.  You should prefer just ``{{version}}`` or ``{{version}}.*`` - the latter is what conda-build will do for you if you omit ``.*`` anyway.

Reply:
Changed to `backports.functools_lru_cache {{ version }}` now.